pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -2,18 +2,20 @@
 title: Hour of Code Teacher Resources
 theme: responsive
 social:
-  "twitter:card": photo
-  "twitter:site": "@codeorg"
-  "twitter:url": "https://<%=request.host%>/hourofcode/overview"
-  "twitter:title": "<%= hoc_s(:hour_of_code) %>"
-  "twitter:description": "<%= hoc_s(:social_hoc2018_global_movement) %>"
-  "twitter:image:src": "<%= 'https://' + request.host + '/images/social-media/code-2018-creativity.png' %>"
-  "og:title": "<%= hoc_s(:hour_of_code) %>"
-  "og:description": "<%= hoc_s(:social_hoc2018_global_movement) %>"
-  "og:image" : "<%= 'https://' + request.host + '/images/social-media/code-2018-creativity.png' %>"
-  "og:image:width": '1200'
-  "og:image:height": '630'
+  twitter:card: photo
```

Comment:
It feels slightly disjointed to leave some of this metadata behind.  Would it be more readable to just move it all to the `:ruby` block below?

Reply:
Yeah, I hear ya. I'm definitely of two minds on this; it doesn't feel great in general that we have these two pretty different ways of setting metadata, but as long as we do have both I think we should be okay with supporting both simultaneously.

Unless you have objections, I think I'd like to leave this as-is for now, and think of the work to move everything into the `:ruby` block as being tied in with work to explore deprecation of the yaml header entirely.

Reply:
Tracking [here](https://codedotorg.atlassian.net/browse/FND-809)